### DISCUSSION TITLE
Error: Cannot find module '@atrilabs/core/lib/reactUtilities'

### DISCUSSION THREAD

**Discussion by Shubhodeep100** [Atri-Labs/atrilabs-engine#680] [Help: Miscellaneous]
![Screenshot 2023-02-13 232526](https://user-images.githubusercontent.com/96099026/218654039-a6d767bd-7a86-4509-afc7-867c1213f55d.jpg)
I'm getting this error on successfully working with the Food-Restaurant website's Homepage. On clicking the 'Build & Run' button, I'm getting the link (localhost:4005) where my app is published, but on visiting the localhost:4005, I'm getting a big error message as show in the picture. I even updated the Atri engine, but still couldn't fix. Please help me resolve this error.

**Comment by cruxcode** [comment]
Don’t worry about Build and Run if it’s not working. We will judge based on how it looks in the editor.
We understand that the assignment is a bit demanding for students, hence, we would like to put off some burden.
Make sure you use correct CSS for positioning and aligning things.
Also make sure components are group together semantically.
These are the parameters on which we will judge.

**Reply by Shubhodeep100** [reply]
Thankyou so much sir, for your response and your supportive consideration towards the technical issue that I'm facing at that particular step.
Based on your response, If you or any other member of the team could please kindly assist me as to what I need to do further in this process; then that'll be very much helpful for me.

**Reply by Selectaman** [reply]
> Don’t worry about Build and Run if it’s not working. We will judge based on how it looks in the editor. We understand that the assignment is a bit demanding for students, hence, we would like to put off some burden. Make sure you use correct CSS for positioning and aligning things. Also make sure components are group together semantically. These are the parameters on which we will judge.

Sir I had completed the assignment in my localhost using Atri Framework editor. How can I show it to you?
